google-basemap/.env

```diff
@@ -0,0 +1,9 @@
+# API Base URL (copy this from CARTO Workspace -> Developers)
+VITE_API_BASE_URL=https://gcp-us-east1.api.carto.com
+# Connection ID (app.carto.com -> Developers -> Manage API Access Tokens)
+VITE_API_CONNECTION_NAME=bqconn2
+# Set this variable if you're using an API Access Token (public application).
+# Go to app.carto.com -> Developers -> Manage API Access Tokens
+VITE_API_ACCESS_TOKEN=eyJhbGciOiJIUzI1NiJ9.eyJhIjoiYWNfN3hoZnd5bWwiLCJqdGkiOiIwMDEzOTI5NCJ9.Y06WtEddE6a-CoGuad8AwMgWV26pC5OowXKxzUOcXIg
```

Comment:
You need to use the same token than other examples. To add your grants, I shared in lastpass the credentials with you. In that account, you will see a token `deck.gl-examples` that's the one you need to edit

Reply:
Done